packages/ui-library/src/components/modal/index.tsx

```diff
@@ -165,7 +169,10 @@ const Modal: React.FC<ModalProps> = ({
       fullWidth
       maxWidth={maxWidth || 'lg'}
       onClose={handleClose}
-      PaperProps={fullHeightProps}
+      PaperProps={{
+        ...fullHeightProps,
+        style: { maxWidth: customMaxWidth },
```

Comment:
so which one is going to take precedence? Is there no case of modal that we can use the maxWidth already provided?